pkg/auth/handler/webapp/as_api_error.go

```diff
@@ -0,0 +1,15 @@
+package webapp
+
+import (
+	"github.com/authgear/authgear-server/pkg/core/skyerr"
+)
+
+func asAPIError(anyError interface{}) *skyerr.APIError {
+	if apiError, ok := anyError.(*skyerr.APIError); ok {
```

Comment:
This check is not needed anymore, since `skyerr.AsAPIError` does this check already.